docs/remote/codespaces.md

```diff
@@ -50,6 +50,38 @@ There are getting started topics for both GitHub Codespaces clients. These will
 
 The VS Code extension API hides most of the implementation details of running remotely so many extensions will just work in GitHub Codespaces environments without any modification. However, we recommend that you test your extension in a codespace to be sure that all of its functionality works as expected. See the article on [Supporting Remote Development and GitHub Codespaces](/api/advanced-topics/remote-extensions.md) for details.
 
+## Known limitations and adaptations
+
+While working with Codespaces and specifically VS Code in the web, there are certain limitations to keep in mind. Some of these limitations have workarounds or adaptations in place to provide a consistent development experience.
+
+| Issue | Reason | Workaround |
+|-|-|-|
+| `Ctrl+Shift+P` won't launch the Command Palette in Firefox. | `Ctrl+Shift+P` is reserved in Firefox. | Use `F1` to launch the Command Palette. |
+| Certain default keybindings (i.e. for debugging) are different in the web. | Since the browser may already have an action registered for those keybindings, we adjust the default for VS Code in the web. | Use the adjusted defaults. They appear on tooltip hovers over the debug actions. <ul><li> Step over is `Alt` + `F10` (instead of `F10`) on all browsers.</li><li> Step into in the web is `Alt` + `F11` (instead of `F11`) on Windows browsers.</li></ul> |
+| `F11` for debugging does not work on macOS in web or desktop.  | This is a known, non-browser specific limitation. More information can be found in [this issue](https://github.com/microsoft/vscode/issues/5102). | Disable `F11` to show desktop on macOS.<ul><li> Go to: System Preferences -> Keyboard -> Shortcuts </li><li> Uncheck the "Show Desktop F11" option </li></ul> |
+| `Ctrl+n` for new file doesn't work in web. | `Ctrl+n` opens a new window instead. | `Ctrl+n` for new file works in the desktop. |
```

Comment:
We may also want to mention `ctrl+w`/`cmd+w`.

Reply:
Great catch - just added.